pkg/components/scheduler.go

```diff
@@ -283,10 +283,18 @@ func (s *scheduler) createInitUserScript() string {
 	return strings.Join(script, "\n")
 }
 
-func (s *scheduler) prepareInitOperationArchive() {
+func (s *scheduler) prepareInitOperationsArchive() {
```

Comment:
Could you please add comment explaining what is the reason for migration here (new script in some new ytsaurus version or same script was renamed or smth)?

Reply:
As far as I see [currently](https://github.com/l0kix2/ytsaurus/blob/main/yt/docker/ytsaurus/Dockerfile#L26) script is called `/usr/bin/init_operation_archive` as in the operator's code. 
Do you plan to change the name of that script in the ytsaurus image in some version?

Reply:
[yes](https://github.com/ytsaurus/ytsaurus/commit/e5348fef221110ce27bba13df5f9790649084b01)

Reply:
I see, then I suggest we write short comment that file was renamed in the image. Also it would be useful to provide this link to commit in the comment.